extra-requirements.txt

```diff
@@ -69,5 +69,6 @@ bs4:                        cicd
 aiostream:                  standard, daemon, devel
 jsonschema:                 cicd
 dgl>=0.6.1:                 devel
+kubernetes==18.20.0b1:      core
```

Comment:
lets remember to remove this from core requirements. We should maybe add a new tag